src/adapter/bun/conninfo.ts

```diff
@@ -0,0 +1,36 @@
+// @denoify-ignore
+import type { Context } from '../..'
+import type { GetConnInfo } from '../../helper/conninfo'
+
+/**
+ * Get ConnInfo with Bun
+ * @param c Context
+ * @returns ConnInfo
+ */
+export const getConnInfo: GetConnInfo = (c: Context) => {
+  const server = ('server' in c.env ? c.env.server : c.env) as
+    | {
+        requestIP?: (req: Request) => {
+          address: string
+          family: string
+          port: number
+        }
+      }
+    | undefined
+
+  if (!server) {
+    throw new TypeError('env has to include 2nd argument of fetch.')
```

Comment:
This is nitpicking but, `env has to include 2nd argument of fetch.` would be

```
env has to include the 2nd argument of fetch.
```

Reply:
I fixed it!